wdl_runner/cromwell_driver.py

```diff
@@ -106,8 +111,10 @@ def submit(
                 break
             except requests.exceptions.ConnectionError as e:
                 logging.info(
-                    "Failed to connect to Cromwell (attempt %d): %s", attempt + 1, e
+                    "Failed to connect to Cromwell (attempt %d of %d): %s",
+                    attempt + 1, max_time_wait // wait_interval, e
                 )
+                logging.info("Sleep for %d seconds before next attempt", wait_interval)
                 time.sleep(wait_interval)
 
         if not job:
```

Comment:
Can we print the entire `job` object in any case? (after this case, a few lines down).

Reply:
Thanks for the suggestion!

if I print the entire `job` object a few lines down on line 124/125, it will look like

```
2022-08-15 17:41:32,154 cromwell_driver INFO: Response: {'id': '17785ee2-6a86-4e15-b0a6-4a7c52477a48', 'status': 'Submitted'}
2022-08-15 17:41:32,155 cromwell_driver INFO: Current job: {'id': '17785ee2-6a86-4e15-b0a6-4a7c52477a48', 'status': 'Submitted'}
2022-08-15 17:41:32,155 cromwell_driver INFO: Job submitted to Cromwell. job id: 17785ee2-6a86-4e15-b0a6-4a7c52477a48
```
which I find a bit more redundant since I already have 
```
logging.info("Response: %s", r.json())
```
on [line 63](https://github.com/deepgenomics/wdl-runner/blob/974427936f7af62074afd7e94b3ebf8dedfea56f/wdl_runner/cromwell_driver.py#L63)

What do you think? or perhaps I should remove line 63 and print current job object? 🤔

Reply:
I agree, it's redundant.